.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18.x'
+      - name: Install dependencies
+        run: npm install
+      - name: Build & archive
+        run: npm run webpack && tar -zcf build.tar.gz build
```

Comment:
I would advocate for an explicit name here: `k6-jslib-aws-0.11.0.tar.gz`. So I know what that archive contains, just by looking at its name when I find it back six months from now in my downloads folder 😄 

What do you think?

Reply:
I see your point about the naming to help discovery in local folders. However, I still prefer to keep the name since it will be the convention between the sub-repositories, and the process of looking for the release build for the future porting to the jslib.k6.io is much simpler and less error-prone.

Reply:
Fair enough 👍🏻 Let's go for that then.

Reply:
>humans using the releases in the process too

I can't add more. That's precisely what I'm trying to do. In the end, the end goal of this process is to align all sub-repositories and improve the delivery of the builds to the `jslib.k6.io` since it's the most straightforward place (and probably) the most convenient for humans.

Reply:
I didn't mean to argue @olegbespalov, I agree the effort to improve `jslib.k6.io` will be beneficial (to humans too ^^) 👍🏻

Reply:
> [...] the process of looking for the release build for the future porting to the jslib.k6.io is much simpler and less error-prone.

I'm happy to keep it as-is, of course, but could you clarify this, please? @olegbespalov 

<br/>
Honestly, I mostly copied the workflow definition from k6-jslib-summary, but now that @oleiade has suggested it, I feel like it could be a fair suggestion to consider. My main thoughts around that are:

- If there's any manual step in the process of publishing it (like creating the PR in [jslib.k6.io](https://github.com/grafana/jslib.k6.io) repo), isn't it less error prone to explicitly see the version you're manipulating the files of?

- If we care about the directory name (I see in [jslib.k6.io](https://github.com/grafana/jslib.k6.io) repo it's just the version number), couldn't we generate a compressed file name like `k6-jslib-aws-0.11.0.tar.gz`, that contains the `0.11.0` folder with all the files?

Thanks! 🙇🏻

Reply:
@oleiade Just in case, I haven't taken this as arguing and didn't argue. I just wanted to provide the complete picture of the idea behind the name.

Reply:
Sounds good I appreciate it 🙇🏻 It's always difficult for me to decipher intentions, tone, and "read the room" in written online discussions, so I prefer to be safe than sorry 🤝

Reply:
@joanlopez for both your thoughts it's true for the current state of the things. 

But the main idea is to get rid of this manual steps. Or at least for beginning script them and end it up with a simple command (in `jslib.k6.io`) that could do most of the routine which are:

1) downloads an asset file of a certain release from the release `https://github.com/grafana/{pkgRepo}/releases/download/{pkgVersion}/build.tar.gz`
2) extracts this under the `lib/{pkgName}/{pkgVersion}`
3) updates `supported.json` and index.html
n) another routine that could be automated

Reply:
Actually, writing things down helps as usual, and I see no issues downloading files from `https://github.com/grafana/{pkgRepo}/releases/download/{pkgVersion}/{pkgRepo}-{pkgVersion}.tar.gz`, and the file generates automatically so if you folks feel that it's improvement let's go this way :+1: